packages/contracts/contracts/test/plugin-mock/UUPSUpgradeable/PluginUUPSUpgradeableMock.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {PluginUUPSUpgradeable} from "../../../core/plugin/PluginUUPSUpgradeable.sol";
+import {IDAO} from "../../../core/IDAO.sol";
+
+contract PluginUUPSUpgradeableV1Mock is PluginUUPSUpgradeable {
+    uint256 public v1;
+
+    function initialize(IDAO _dao) external virtual initializer {
+        __PluginUUPSUpgradeable_init(_dao);
+        v1 = 1;
+    }
+}
+
+contract PluginUUPSUpgradeableV2Mock is PluginUUPSUpgradeableV1Mock {
+    uint256 public v2;
```

Comment:
Having natspecs, at least a comment or clear naming, convey better the purpose of the state variable and the mock

Reply:
True, I renamed it to `stateX`